src/server/models/user.js

```diff
@@ -21,8 +21,17 @@ const userSchema = new Schema({
         type: String,
         required: true
     }
-
-
 });
 
-module.exports = mongoose.model("PublicUser", userSchema);
\ No newline at end of file
+// user.js
```

Comment:
Once the front-end sign up page is created, we can remove the commented lines.

Reply:
Will do